src/components/ApplicationMenu/ApplicationMenu.js

```diff
@@ -77,6 +81,10 @@ class ApplicationMenu extends Component<Props> {
             label: 'Getting Started',
             click: this.openGettingStartedDocs,
           },
+          {
+            label: 'Report an issue',
```

Comment:
You'll want to capitalize the `i` to match existing menu options (leaving the `an` lowercase is good): `'Report an Issue'`.

Reply:
Ah yes good catch. Btw, I've seen the windows PR and on windows, the other words are not capitalized.

Reply:
Is that just a quirk of Electron on Windows or do they need to be fixed? I didn't see any changes to `ApplicationMenu.js` in #125.

Reply:
It will need to be fixed yes